part2/set1/1/numbers/opnotequal.cc

```diff
@@ -0,0 +1,8 @@
+#include "numbers.ih"
+
+    // by 
+
+bool operator!=(Numbers const &lhs, Numbers const &rhs) 
+{
+    return not (lhs == rhs);
```

Comment:
```suggestion
    return !(lhs == rhs);
```

It's slightly unconventional to use the `not` keyword, `!` is more common.